startup/gui/menus.py

```diff
@@ -536,6 +536,18 @@ def __usdShaderCreator( shaderName ) :
 ] :
 	nodeMenu.append( "/USD/Shader/{}".format( menuPath ), functools.partial( __usdShaderCreator, shader ), searchText = shader )
 
+def __usdLightCreator( lightType ) :
+
+	light = GafferUSD.USDLight( name = lightType )
+	light["name"].setValue( lightType[0].lower() + lightType[1:] )
+	light.loadShader( lightType )
+	return light
+
+for lightType in [
+	"DistantLight", "DiskLight", "RectLight", "SphereLight", "CylinderLight", "DomeLight",
+] :
+	nodeMenu.append( "/USD/Light/{}".format( IECore.CamelCase.toSpaced( lightType ) ), functools.partial( __usdLightCreator, lightType ), searchText = lightType )
```

Comment:
Might be worth adding a convenience for creating a SpotLight from the tab menu? It'd only save a few steps but I'm sure users would appreciate it and it would make the creation of spotlights a bit more discoverable...

Reply:
Good call - added in https://github.com/GafferHQ/gaffer/pull/5400/commits/fb7ffada7b11e8f10061e15620cf4d63b74c98ac. I wasn't sure what to do about the radius, which I've left at default. Maybe something smaller would be better?